src/helper/directory_helper.rs

```diff
@@ -165,6 +172,7 @@ impl DirectoryHelper {
     /// Creates the directory or the root or both if it doesn't find one.
     #[allow(dead_code)]
     pub fn get_configuration_directory_listing(&self, directory_name: String) -> Result<::directory_listing::DirectoryListing, ::errors::NfsError> {
+        debug!("Retrieving directory listing configurations ...");
```

Comment:
Also please log the parameters or variables that would be good while debugging. 

For eg the message here can be:
debug!("Retrieving a MaidSafe-Specific-Configuration directory listing for {:?}",  directory_name)


also note that there are few more places in the same function where the logs would add value. There is a branching statement which either creates and returns the directory or returns an existing directory. Adding a log would be quite useful to debug such scenarios.

Do have a read through and add the debug logs in places that would be helping us more.